content/products/hosted/notable-features.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Hosted Dolt: Notable Features"
+---
+
+# Version control
+
+[Dolt](https://doltdb.com) is a database like no other. There are numerous Dolt version
+control features that you won't find elsewhere. For example:
+
+- Create branches of your data.
+- Merge changes you make with changes made by other people, and resolve conflicts using SQL.
+- Look at the state of the data a given commit.
+- Diff data between two commits or branches.
+- Revert the changes made by a commit.
+
+# Logs
```

Comment:
Maybe group these?

# Operations
## Access Management
## Logs
## Monitoring
## Backups
## Replication
## Dolt Upgrades
## Enterprise Support

# SQL Workbench
## User Friendly Web GUI
## Pull Requests
## Commit Log
## ORM Diagrams

# Dolt Ecosystem
## Clone a Hosted Instance
## Use DoltHub as a Remote

I think you get it.